### DISCUSSION TITLE
Styling of inbuild drag and drop

### DISCUSSION THREAD

**Discussion by CodeFromTheEmpty** [KevinVandy/material-react-table#1234] [Q&A]
Dear Community,

we use Material React Table v2 and the inbuild drag and drop system for row ordering.

We have styled the table itself to our needs. But I cant find where to style the drag and drop appearance. We couldn't find anything in the mrt documentation or google.

Does anybody have a hint for us, where to look further?

Thx in advance for your help!

https://stackoverflow.com/questions/78948913/material-react-table-styling-of-drag-and-drop

**Comment by KevinVandy** [comment]
Just would have to inspect element or look at the source code for this

**Reply by CodeFromTheEmpty** [reply]
Hi Kevin, thanks alot for your time and answer.
Does this mean, that we just should find the css classes and target them from outside the table with a css-sheet?
All the other table-styling was done in tableoptions, redefining render functions etc.
examples:
renderEmptyRowsFallback
muiSearchTextFieldProps
muiTableBodyRowProps

But styling of DnD has to be done from "outside"?

**Reply by KevinVandy** [reply]
No, probably has to be done through inline styles if anything

**Reply by CodeFromTheEmpty** [reply]
Thx, the problem is i have no idea, where to add these styles. Is there any example or codesandbox with styled DnD on the web?

**Reply by KevinVandy** [reply]
The muiProps column/table options.

**Reply by CodeFromTheEmpty** [reply]
Thanks again.
It does'nt feel good to not be able to understand the help you provide. I am very sorry for that..

If I look at the MuiProps directly in Mui as recommend. There are no api informations for DnD, dragStart, end over etc.
If i look at the MuiProps used by MRT_Table (such as MuiSearchTextFieldProps, which we used to style the inputfield for the global search) i cant find any Props directed at DnD.

I am very sorry, propably I am just missing the point here.

**Reply by KevinVandy** [reply]
Like the muiTableBoyCellProps and the like

**Reply by CodeFromTheEmpty** [reply]
I am very sorry, that I dont get it.

**Comment by CodeFromTheEmpty** [comment]
Thanks again, I do understand that,and we used these extensively, but i have no clue how to target drag and drop stylings there. I think just one example would be sufficient to understand the syntax to figure out the rest myself.

**Comment by itshallrun** [comment]
@CodeFromTheEmpty Have you found a solution to this?

**Reply by CodeFromTheEmpty** [reply]
Unfortunately not. I have delayed this task and hoped to get a hint. But nothing I can add at the moment.
